File: src/main/java/com/server/computerscience/chatbot/dto/response/ChatBotResponseDto.java
package com.server.computerscience.chatbot.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;

@AllArgsConstructor
@Getter
@Builder
public class ChatBotResponseDto {
	private String content;

	public static ChatBotResponseDto from(String content) {
		return ChatBotResponseDto.builder().content(content).build();
	}
}

File: src/main/java/com/server/computerscience/chatbot/dto/response/ChatBotResponseDto.kt
package com.server.computerscience.chatbot.dto.response


class ChatBotResponseDto(
    val content: String? = null
) {

    companion object {
        fun from(content: String?): ChatBotResponseDto {
            return ChatBotResponseDto(
                content = content
            )
        }
    }
}
